PWG/EMCAL/EMCALbase/AliAnalysisTaskEmcal.cxx

```diff
@@ -1294,6 +1235,57 @@ Bool_t AliAnalysisTaskEmcal::IsEventSelected()
   return kTRUE;
 }
 
+Bool_t AliAnalysisTaskEmcal::IsTriggerSelected(){
+  // Default implementation of trigger selection
+  // same as previously (code moved from IsEventSelected
+  // to trigger selection). Users should re-implement
+  // this function in case they have certain needs, in
+  // particular for EMCAL triggers
+  if (!fTrigClass.IsNull()) {
+    TString fired = InputEvent()->GetFiredTriggerClasses();
+    if (!fired.Contains("-B-")) return kFALSE;
+
+    std::unique_ptr<TObjArray> arr(fTrigClass.Tokenize("|"));
+    if (!arr) return kFALSE;
+    Bool_t match = 0;
```

Comment:
I know you're just moving around the code, but since we're looking at this, can we change the values that we are setting in this bool to `true` and `false` (as appropriate)? I think it's preferable to use the values associated with the type.

Reply:
Sure, thanks for spotting. I anyhow wanted to fix this but then forgot about this. I will add it to the commit.